BitGo/BitGoJS
# fix(key-card): fix errors in key-card drawing code

* adds more examples for key cards to download to the web-demo module
* fixes bugs discovered through these examples

Ticket: BG-64432

## Test instructions

1. Check out this branch locally
2. Run `yarn install` in the root of the repository
3. `cd modules/web-demo`
4. `yarn dev`
5. Go to http://localhost:8080/keycard
6. Download all three types of key cards and compare to the ones attached to this PR.

[BitGo Keycard for Hello World - polygon.pdf](https://github.com/BitGo/BitGoJS/files/10705911/BitGo.Keycard.for.Hello.World.-.polygon.pdf)
[BitGo Keycard for Hello World - ethereum.pdf](https://github.com/BitGo/BitGoJS/files/10705914/BitGo.Keycard.for.Hello.World.-.ethereum.pdf)
[BitGo Keycard for Hello World - litecoin.pdf](https://github.com/BitGo/BitGoJS/files/10705916/BitGo.Keycard.for.Hello.World.-.litecoin.pdf)
